dbt/dbt_project.yml

```diff
@@ -37,14 +36,6 @@ vars:
   # frames for testing
   test_qc_year_end: 2030
 
-  # Locations in S3 where Python model dependencies are stored. Since these
-  # locations vary by target, it would be nice if we could define them as target
-  # attributes in profiles.yml, but we can't add new profiles.yml attributes
-  # without extending dbt-athena so we define the locations here instead
-  s3_dependency_dir_dev: s3://ccao-dbt-athena-dev-us-east-1/dependencies
-  s3_dependency_dir_ci: s3://ccao-dbt-athena-ci-us-east-1/dependencies
-  s3_dependency_dir_prod: s3://ccao-athena-dependencies-us-east-1
-
```

Comment:
Also dropping these now that the package location is global and hard-coded in the deps script.